### PR TITLE
cmocka 1.0.0

### PR DESCRIPTION
**Pull request by mythosil**

### CODE REVIEW

File: Library/Formula/cmocka.rb [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
 
   def install
-    mkdir 'build' do
+    mkdir "build" do
       system "cmake", "..", "-DUNIT_TESTING=On", *std_cmake_args
       system "make"
-      system "make install"
+      system "make", "install"
     end
   end
```
Comment (mikemcquaid) [inline Library/Formula/cmocka.rb]: Any chance a test can be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by mythosil** [comment]
Sorry for my bad manner. I just added a test.

**Comment by mikemcquaid** [comment]
No bad manners at all, this was great as is but just thought I'd ask. Thanks again!

File: Library/Formula/cmocka.rb [hunk 2]
```diff
@@ -1,17 +1,37 @@
...
+
+      int main(void) {
+        const struct CMUnitTest tests[] = {
+            cmocka_unit_test(null_test_success),
+        };
+        return cmocka_run_group_tests(tests, NULL, NULL);
+      }
+    EOS
+    system ENV.cc, "test.c", "-I#{include}", "-L#{lib}", "-lcmocka", "-o", "test"
+    system "./test"
```
Comment (mikemcquaid) [inline Library/Formula/cmocka.rb]: :heart_eyes:

**Comment by tdsmith** [comment]
@BrewTestBot test this please

**Comment by tdsmith** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**